scalafix-core/src/main/scala/scalafix/v1/SymbolProperties.scala

```diff
@@ -21,4 +24,28 @@ final class SymbolProperties private[scalafix] (props: Int) {
   // privates
   private[this] def is(property: s.SymbolInformation.Property): Boolean =
     (props & property.value) != 0
+
+  override def toString: String = {
+    val buf = ListBuffer.empty[String]
+    if (isAbstract) buf += "abstract"
+    if (isFinal) buf += "final"
+    if (isSealed) buf += "sealed"
+    if (isImplicit) buf += "implicit"
+    if (isLazy) buf += "lazy"
+    if (isCase) buf += "case"
+    if (isContravariant) buf += "covariant"
+    if (isContravariant) buf += "contravariant"
+    if (isStatic) buf += "static"
+    if (isPrimary) buf += "primary"
+    if (isEnum) buf += "enum"
+    if (isVal) buf += "val"
+    if (isVar) buf += "var"
+    buf.mkString("SymbolProperties(", " ", ")")
+  }
```

Comment:
I wonder if we can use metap for that.

Reply:
We can, but it requires making metap more flexible: https://github.com/scalameta/scalameta/pull/1751